src/graphback-core/index.ts

```diff
@@ -0,0 +1,57 @@
+import { GraphQLBackendCreator, DatabaseSchemaManager } from 'graphback';
+import { transpile } from "typescript";
+import * as folders from './config.json';
+
+export async function generateTypeDefs(model: string, config: {[id: string]: any}) {
+  const creator = new GraphQLBackendCreator(model, config);
+  const backend = await creator.createBackend("sqlite3");
```

Comment:
We should perform this operation once and reuse the result for `generateTypeDefs`, `generateResolvers` and `generateDatabase`

Reply:
You're right, I'm repeating the code in all three methods :sweat_smile:

Reply:
@b1zzu Would it be a good solution to encapsulate the methods of this file in a class ? By this approach, we can receive the variables _model_ and _config_ as properties (and reuse them when necessary) and perform the operations that you've marked once in the constructor (saving the results in private properties to use in the class methods), wdyt ?

Reply:
I think it's a good idea, we have to see than in practice if it works good, but I wouldn't focus too much on this if we can't get anything easy and fast we should create a separate issue and handle it later on

Reply:
@vinifarias focus first on providing the db